scipy/stats/tests/test_mstats_basic.py

```diff
@@ -966,19 +966,6 @@ def test_signaltonoise(self):
                 rm = stats.mstats.signaltonoise(ym)
                 assert_almost_equal(r, rm, 10)
 
-    def test_betai(self):
```

Comment:
You don't want to delete the tests just yet (they're useful until we remove ``betai`` from the code), but instead just silence the ``DeprecationWarning``